src/main/java/org/opensearch/ad/rest/handler/AbstractAnomalyDetectorActionHandler.java

```diff
@@ -608,10 +608,10 @@ protected void validateCategoricalField(String detectorId, boolean indexingDryRu
         ActionListener<GetFieldMappingsResponse> mappingsListener = ActionListener.wrap(getMappingsResponse -> {
             // example getMappingsResponse:
             // GetFieldMappingsResponse{mappings={server-metrics={_doc={service=FieldMappingMetadata{fullName='service',
-            // source=org.opensearch.common.bytes.BytesArray@7ba87dbd}}}}}
+            // source=org.opensearch.core.common.bytes.BytesArray@7ba87dbd}}}}}
```

Comment:
This change is commented out. Is this for future use?

Reply:
It is a comment. The change is updating the comment.